Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            Program.HW1();
+            Console.WriteLine("Okay, Person #2:");
+            Program.HW1();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
+        {
+            //Function Declarations
+            double BMI, hSquared;
+
+            //convert cm to meters and square
+            hSquared = (height / 100d);
+            hSquared *= hSquared;
+
+            BMI = weight / hSquared;
```

Comment:
You can directly `return weight / hSquared` . Because the function name is GetBMI (very good name, by the way), your extra variable doesn't add extra value.